src/test/kotlin/blackjack/domain/BlackjackGameTest.kt

```diff
@@ -0,0 +1,101 @@
+package blackjack.domain
+
+import blackjack.ui.model.PlayerDto
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+internal class BlackjackGameTest {
+
+    @Test
+    fun `모든 player에게 카드를 준다`() {
+        val players = Players(mutableListOf(Player("song"), Player("kim")))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val players = Players(listOf(Player("song"), Player("kim")))
```